stablehlo/reference/Element.cpp

```diff
@@ -619,6 +646,35 @@ Element areApproximatelyEqual(const Element &e1, const Element &e2) {
                                      debugString(type).c_str()));
 }
 
+Element areClose(const Element &e1, const Element &e2,
```

Comment:
Being different overloads we can keep the same name as `areApproximatelyEqual`?

Reply:
Yes, now using `areApproximatelyEqual` with optional `abs_error` argument and `areClose` is removed.